skyportal/models.py

```diff
@@ -322,6 +322,13 @@ class Obj(Base, ha.Point):
 
     origin = sa.Column(sa.String, nullable=True, doc="Origin of the object.")
 
+    internal_key = sa.Column(
+        sa.String,
```

Comment:
I thought we converged on `int`s? I guess it doesn't really matter, but may be easier to pass ints around for debugging purposes.

Reply:
I never heard anyone weigh in on strings vs ints. Do you want me to change this to int?

Reply:
@dannygoldstein SQLAlchemy does not support auto-incrementing non-primary key int fields; so we'd be using a (large) random int instead of a UUID string, which I don't think actually makes debugging any easier (if they were small ints, I could see the case for that)

Reply:
Saw your slack comment; resolving this